src/chi-vue/src/components/column-customization/AvailableColumns.tsx

```diff
@@ -7,6 +7,7 @@ import { UTILITY_CLASSES } from '@/constants/classes';
 @Component
 export default class ColumnCustomizationAvailableColumns extends Vue {
   @Prop() availableColumns?: DataTableColumn[];
+  @Prop() handleSelectColumn?: (column: DataTableColumn) => void;
```

Comment:
Instead of receiving a handler, this component should emit an event, and the parent component should listen to that event and handle the column selection. That is the common front-end applications architecture.